addons/account_check_printing/account_payment.py

```diff
@@ -30,9 +32,12 @@ def _onchange_amount(self):
         # TODO: merge, refactor and complete the amount_to_text and amount_to_text_en classes
         check_amount_in_words = amount_to_text_en.amount_to_text(math.floor(self.amount), lang='en', currency='')
         check_amount_in_words = check_amount_in_words.replace(' and Zero Cent', '') # Ugh
-        decimals = self.amount % 1
```

Comment:
`decimals = round(self.amount % 1, 2)` and you don't need to add extra libraries and code. What do you think?